marlowe-playground-server/src/Server.hs

```diff
@@ -84,13 +109,26 @@ app handlers =
     policy =
       simpleCorsResourcePolicy
 
-newtype AppConfig = AppConfig {authConfig :: Auth.Config}
+data AppConfig = AppConfig { authConfig :: !Auth.Config, webghcClientEnv :: !ClientEnv }
 
-initializeServerContext :: Maybe FilePath -> IO AppConfig
-initializeServerContext secrets = do
+initializeServerContext :: Second -> Maybe FilePath -> IO AppConfig
+initializeServerContext maxInterpretationTime secrets = do
   putStrLn "Initializing Context"
   authConfig <- mkAuthConfig secrets
-  pure $ AppConfig authConfig
+  mWebghcURL <- lookupEnv "WEBGHC_URL"
+  webghcURL <- case mWebghcURL of
+    Just url -> parseBaseUrl url
+    Nothing -> do
+      let localhost = "http://localhost:8009"
```

Comment:
It seems this needs to be `http://localhost:8080`, because it is this service that is deploying `webghc` when deployed locally, not sure how that happens, but I tested

Reply:
https://github.com/input-output-hk/plutus/pull/3741